src/extension/background-script/migrations/index.ts

```diff
@@ -52,6 +52,19 @@ const migrations = {
       // state is saved with the setMigrated call
     }
   },
+  migrateisUsingGlobalNostrKey: async () => {
+    const { nostrPrivateKey, accounts } = state.getState();
+
+    if (nostrPrivateKey) {
+      Object.values(accounts).map((account) => {
+        account.nostrPrivateKey = nostrPrivateKey;
```

Comment:
let's make this defensive and only overwrite it when`account.nostrPrivateKey` is blank.